devel/usbredir/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                usbredir
+version             0.8.0
+maintainers         {danchr @danchr} openmaintainer
+categories          devel
+platforms           darwin
+license             BSD
+description         Remote virtual machine server
+long_description    {*}${description}
+
+homepage            https://www.spice-space.org/usbredir.html
+master_sites        https://www.spice-space.org/download/usbredir/
+
+use_bzip2           yes
+
+checksums           rmd160  ff51d678e7e3b5d0bb3d7d16335349143e063c3b \
+                    sha256  87bc9c5a81c982517a1bec70dc8d22e15ae197847643d58f20c0ced3c38c5e00 \
+                    size    331349
+
+patch.pre_args      -p1
+patchfiles          sol_tcp.diff
+
+depends_build       port:pkgconfig
+
+depends_lib         port:libusb
```

Comment:
libusb is available as a -devel port so please rewrite this dependency in `path:` style. There seems not to have been a concerted effort to do this in other ports that depend on libusb; I've filed https://trac.macports.org/ticket/60689 for that.

Reply:
I'll just leave it as-is for now, as that's just as easily done as part of an automated change 🙂